### DISCUSSION TITLE
Friend Banned

### DISCUSSION THREAD

**Discussion by RoyTheReal** [Sketchware-Pro/Sketchware-Pro#874] [General]
Hey, my friend got banned from this GitHub and I was wondering if you could please unban him. His sister got on his account and started messing with stuff and he somehow got banned so please help him out. His account is called Fooks Technology.

**Comment by nicesapien-too** [comment]
I dont think its possible to ban someone from repo, try contacting microsoft

**Comment by j-px** [comment]
Of course it's just his sister. What sister would know to create a (bad try at a) DoS app, [create a false bug report](https://github.com/Sketchware-Pro/Sketchware-Pro/issues/865), and upload the .apk file to Firebase Storage, while being childish enough to do that "from her brother's account"? Creating this discussion with a new account also seems suspicious.

**Reply by RoyTheReal** [reply]
No, this is all my friend has told me I swear. I'm sorry if I caused any inconvenience. I didn't know that all that happened, his sister is only 8 so uhm yeah I don't think she would know to do that hehe
